libcalico-go/lib/epstatusfile/status_file_watcher.go

```diff
@@ -80,6 +81,14 @@ func (w *FileWatcher) Start() {
 }
 
 func (w *FileWatcher) newFsnotifyWatcher() error {
+	oldWatcher := w.fsWatcher
```

Comment:
This also gets closed by the defer in `runWatcher()`, might end up closing it twice.